src/shell/navigation/Navigation.tsx

```diff
@@ -11,44 +15,39 @@ interface INavigationState {
     title: string
 }
 
-export class Navigation extends React.Component<INavigationProps, INavigationState> {
-
-    public state = {
-        navigationExpanded: false,
-        title: 'Navigation'
-    }
+const navigation = props => {
```

Comment:
you have to add types for fuctional component. Actually VS code shows you proplems with types and you should deal with it. 
hint:
const navigation: FunctionComponent<INavigationProps> = props => ...

Reply:
Got it.